Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteTextureProxy.cpp

```diff
@@ -44,24 +44,24 @@ RemoteTextureProxy::RemoteTextureProxy(RemoteGPUProxy& root, ConvertToBackingCon
 
 RemoteTextureProxy::~RemoteTextureProxy()
 {
-    auto sendResult = send(Messages::RemoteTexture::Destruct());
-    UNUSED_VARIABLE(sendResult);
+    if (m_createdView) {
+        auto sendResult = send(Messages::RemoteTexture::Destruct());
```

Comment:
why do we only call destruct if `m_createdView` is true?

Reply:
I see the comment in the commit log but I don't completely follow:
> Messages::RemoteTexture::Destroy was being sent without
   Messages::RemoteTexture::CreateView having been sent
but the texture is created from RemoteDevice::CreateTexture, not RemoteTexture::CreateView